### PR TITLE
Start french translation

### PR DESCRIPTION
**Pull request by Natim**
I am not sure how you want to handle PR. Maybe we should create a français branch?

### CODE REVIEW

**Comment by ametaireau** [comment]
+1 for a french branch here so we can merge everything there.

File: README.rst [hunk 1]
```diff
@@ -2,33 +2,33 @@
...
 
-#. Installing Python and setting up development environment
-#. Python basics
-#. "Snakes!" online competition
-#. Creating a polling application in Django
-#. Deploying the application
+#. Installer Python et mettre en place son environnement de développement
+#. Les bases de Python
+#. "Snakes!" competition en ligne
+#. Mettre en place une appli de sondage en Django
```
Comment (ametaireau) [inline README.rst]: application

File: README.rst [hunk 2]
```diff
@@ -2,33 +2,33 @@
...
+#. Déployer son application
 
 
-Build instructions
-==================
+Générer la documentation
+========================
 
-Just run::
+Lancer simplement::
```
Comment (ametaireau) [inline README.rst]: Lancez

File: README.rst [hunk 3]
```diff
@@ -2,33 +2,33 @@
...
 
     $ pip install -r requirements.txt
     $ make html
 
-Documentation should be created in ``build/html`` directory.
+La document sera créée dans le répertoire ``build/html``.
```
Comment (ametaireau) [inline README.rst]: créé

File: README.rst [hunk 4]
```diff
@@ -37,22 +37,21 @@ Unicode, go figure)::
     $ xelatex workshops.tex
 
 
-Why Python 3?
-=============
+Pourquoi avoir choisi Python 3 ?
+================================
+
+Parce que Python 3, c'est le futur. Et aussi pour la gestion native de
+l'Unicode ce qui permet de sauter toute la formation à l'Unicode qui
```
Comment (ametaireau) [inline README.rst]: on mets un U majuscule à unicode?

File: README.rst (hunk 1 above)
Comment (Exirel) [inline README.rst]: "Mettre en place" is more "Install". I would suggest a verb like "Créer" or "Développer".

File: source/code_of_conduct.rst [hunk 5]
```diff
@@ -2,32 +2,53 @@
...
-have any other concerns, please act to intercede or ask for help from any member of Geek Girls
-Carrots, IRC Administrators, website Administrators, or Organizers/Representatives of any physical
-events put on under the auspices of Geek Girls Carrots.
-
-Portions of this document have been adapted from the PyLadies Code of Conduct, which was in
-turn adapted from the Plone Foundation Code of Conduct; this particular Code of Conduct has been
-expanded and edited to suit the specific needs of Geek Girls Carrots.
+Geek Girls Carrots s'engage à mettre en place une communauté
+respectueuse pour chacune et chacun. Nous ne tolérons ni le
+harcélement, ni l'intimidation ou la brutalisation de quelque sorte
```
Comment (ametaireau) [inline source/code_of_conduct.rst]: harcèlement

File: README.rst (hunk 2 above)
Comment (Exirel) [inline README.rst]: s/Lancer simplement/En deux lignes::

File: README.rst (hunk 3 above)
Comment (arroway) [inline README.rst]: "La documentation" ou "le document" ?

File: source/code_of_conduct.rst [hunk 6]
```diff
@@ -2,32 +2,53 @@
...
+qu'elle soit sur l'un des membres de la communauté.
+
+Cela ne concerne pas uniquement les membres de GGC mais s'applique à
+toute personne en relation avec les membres, développeurs, intégrateurs
+de communauté Geek Girls Carrots au sens large, au cours des événements
+ou au cours de toute autre interraction.
```
Comment (ametaireau) [inline source/code_of_conduct.rst]: interaction

File: source/code_of_conduct.rst [hunk 7]
```diff
@@ -2,32 +2,53 @@
...
+
+Le harcélement inclu des propos blessants, par voie verbale ou
```
Comment (ametaireau) [inline source/code_of_conduct.rst]: harcèlement

Comment (ametaireau) [inline source/code_of_conduct.rst]: inclus

File: README.rst [hunk 8]
```diff
@@ -37,22 +37,21 @@ Unicode, go figure)::
     $ xelatex workshops.tex
 
 
-Why Python 3?
-=============
+Pourquoi avoir choisi Python 3 ?
```
Comment (Exirel) [inline README.rst]: Short version is fine to me: "Pourquoi Python 3 ?".

File: source/code_of_conduct.rst [hunk 9]
```diff
@@ -2,32 +2,53 @@
...
+éléctronique, à propos des choix ou des caractéristiques d'une
+personne; des images ou des commentaires sexuels en public ou sur des
+espaces en ligne; l'intimidation délibérée, la brutalisation, la
+traque, le suivi d'une personne, la prise de vue photographique ou
+vidéo sans l'accord de la personne, des propos perturbateurs de
+manière verbale, sur IRC (Internet Relay Chat), lors de rendez-vous
+téléphonique, lors de réunions ou lors de tout autre événement; des
```
Comment (ametaireau) [inline source/code_of_conduct.rst]: téléphoniques

File: source/code_of_conduct.rst [hunk 10]
```diff
@@ -2,32 +2,53 @@
...
+
+Les participants auxquels est demandé l'arrêt de ce type de
+comportement sont priés de s'exécuter immédiatement.
+
+Si un participant commence à pratiquer ce genre de comportement, les
+représentants de la communauté pourrons mettre en place les actions
+qu'elle juge nécessaire, incluant un avertissement du contrevenant,
+l'expulsion des événements de Girls Carrots ou l'expulsion des
+mailings lists, conversations IRC, groupes de travail ou de tout autre
+moyen de communication éléctronique afin de résoudre le problème. Cela
```
Comment (ametaireau) [inline source/code_of_conduct.rst]: électronique

File: README.rst (hunk 4 above)
Comment (arroway) [inline README.rst]: je crois qu'il y a bien une majuscule à Unicode
Je mettrais "pourquoi choisir Python 3", plus direct.

File: source/code_of_conduct.rst [hunk 11]
```diff
@@ -2,32 +2,53 @@
...
+peut même aller jusqu'à la radiation de la communauté si cela s'avère
+nécessaire.
+
+Si vous êtes harcelé(e), observez que quelqu'un est harcelé(e) ou avez
+n'importe quel autre soucis, merci d'intervenir et de prendre contact
+avec un membre de GGC, un administrateur IRC, un administrateur du
+site internet, un des organisateurs/représentants lors d'un événement
+sous la responsabilité de GGC.
+
+Une partie de ce document a été adapté du Code of Conduct des PyLadies
```
Comment (ametaireau) [inline source/code_of_conduct.rst]: code de conduite

File: README.rst [hunk 12]
```diff
@@ -2,33 +2,33 @@
 Django Carrots tutorials
 ========================
 
-Tutorials walking new programmers through the process of building
-their first website in Python and Django.
+Le tutoriel Django Carrots présente pas-à-pas aux débutants la manière
+de créer son premier site internet avec Python et Django
```
Comment (arroway) [inline README.rst]: de créer "leur" premier site

File: README.rst [hunk 13]
```diff
@@ -2,33 +2,33 @@
...
+En quelques mots:
 
-#. Installing Python and setting up development environment
-#. Python basics
-#. "Snakes!" online competition
-#. Creating a polling application in Django
-#. Deploying the application
+#. Installer Python et mettre en place son environnement de développement
+#. Les bases de Python
+#. "Snakes!" competition en ligne
```
Comment (arroway) [inline README.rst]: accent aigu sur "compétition

File: source/code_of_conduct.rst [hunk 14]
```diff
@@ -2,32 +2,53 @@
 Geek Girls Carrots Code of Conduct
 ==================================
 
+Geek Girls Carrots (aussi désigné par l'acronyme GGC) recherche à
```
Comment (arroway) [inline source/code_of_conduct.rst]: "cherche à"

File: source/code_of_conduct.rst [hunk 15]
```diff
@@ -2,32 +2,53 @@
...
+favoriser un environnement d'apprentissage encourageant et engageant
```
Comment (arroway) [inline source/code_of_conduct.rst]: "accompagné" plutôt qu'"engageant" ?

File: source/code_of_conduct.rst [hunk 16]
```diff
@@ -2,32 +2,53 @@
...
+dans lequel les individus de tous horizons peuvent participer
+confortablement, sans distinction de race, couleur, origine, age,
+religion, opinion politique, capacité physique, orientation sexuelle,
+sexe ou status socio-economique.
```
Comment (arroway) [inline source/code_of_conduct.rst]: je ne vois pas le mot dans le texte original qui est traduit par "sexe"

File: source/code_of_conduct.rst [hunk 17]
```diff
@@ -2,32 +2,53 @@
...
+harcélement, ni l'intimidation ou la brutalisation de quelque sorte
+qu'elle soit sur l'un des membres de la communauté.
+
+Cela ne concerne pas uniquement les membres de GGC mais s'applique à
+toute personne en relation avec les membres, développeurs, intégrateurs
+de communauté Geek Girls Carrots au sens large, au cours des événements
+ou au cours de toute autre interraction.
+
+Le harcélement inclu des propos blessants, par voie verbale ou
+éléctronique, à propos des choix ou des caractéristiques d'une
```
Comment (arroway) [inline source/code_of_conduct.rst]: électronique

File: source/code_of_conduct.rst [hunk 18]
```diff
@@ -2,32 +2,53 @@
...
+personne; des images ou des commentaires sexuels en public ou sur des
+espaces en ligne; l'intimidation délibérée, la brutalisation, la
+traque, le suivi d'une personne, la prise de vue photographique ou
```
Comment (arroway) [inline source/code_of_conduct.rst]: la prise de photos ou de vidéos sans l'accord de la personne

File: source/code_of_conduct.rst [hunk 19]
```diff
@@ -2,32 +2,53 @@
...
+manière verbale, sur IRC (Internet Relay Chat), lors de rendez-vous
+téléphonique, lors de réunions ou lors de tout autre événement; des
+contacts physiques innapropriés ou des attouchements sexuels non
+sollicitées.
+
+Les participants auxquels est demandé l'arrêt de ce type de
+comportement sont priés de s'exécuter immédiatement.
+
+Si un participant commence à pratiquer ce genre de comportement, les
+représentants de la communauté pourrons mettre en place les actions
```
Comment (arroway) [inline source/code_of_conduct.rst]: pourront

File: source/code_of_conduct.rst [hunk 20]
```diff
@@ -2,32 +2,53 @@
...
+qu'elle juge nécessaire, incluant un avertissement du contrevenant,
```
Comment (arroway) [inline source/code_of_conduct.rst]: qu'ils jugent

File: source/code_of_conduct.rst [hunk 21]
```diff
@@ -2,32 +2,53 @@
...
+Le harcélement inclu des propos blessants, par voie verbale ou
+éléctronique, à propos des choix ou des caractéristiques d'une
+personne; des images ou des commentaires sexuels en public ou sur des
+espaces en ligne; l'intimidation délibérée, la brutalisation, la
+traque, le suivi d'une personne, la prise de vue photographique ou
+vidéo sans l'accord de la personne, des propos perturbateurs de
+manière verbale, sur IRC (Internet Relay Chat), lors de rendez-vous
+téléphonique, lors de réunions ou lors de tout autre événement; des
+contacts physiques innapropriés ou des attouchements sexuels non
+sollicitées.
```
Comment (arroway) [inline source/code_of_conduct.rst]: sollicités

File: source/code_of_conduct.rst (hunk 16 above)
Comment (Natim) [inline source/code_of_conduct.rst]: Il n'y est pas c'est voulu?

**Comment by Natim** [comment]
You guys rock !

**Comment by plocharz-9livesdata** [comment]
Great work, I'll check this out.

French translation will be put in seperate branch: "french". Master branch is designed to english translation (as this is the source language for the scenario). In future we hope to make the translation process easier.

**Comment by qrees** [comment]
Great work, I'll check this out.

French translation will be put in seperate branch: "french". Master branch is designed to english translation (as this is the source language for the scenario). In future we hope to make the translation process easier.

**Comment by ametaireau** [comment]
@Natim I'm not sure if you can change the merge target directly on the pull request?

**Comment by Natim** [comment]
No I will need @qrees to do it. You cannot put a target on a non existing branch and I am not a Collaborator of the repository so I cannot create it.

**Comment by Natim** [comment]
I have merged it in https://github.com/Natim/django-carrots/tree/french for now, I will create new PR on top of it.

**Comment by plocharz-9livesdata** [comment]
Don't worry, you don't have to change merge this and take care of this. Feel free to push new updates and pull requests.

**Comment by qrees** [comment]
Don't worry, you don't have to change merge this and take care of this. Feel free to push new updates and pull requests.

**Pull request closed without merging**